parse/DoubleComplexValueRefParser.cpp

```diff
@@ -26,6 +26,13 @@ namespace parse {
                   )     [ _val = new_<ValueRef::ComplexVariable<double> >(_a, _b, _c, _f, _d, _e) ]
                 ;
 
+            empire_meter_value
```

Comment:
Would be preferable to be able to write `EmpireDetection id = Source.Owner` or `Detection empire = Source.Owner` or similar. The part_capacity example refers to the meters directly, not `ShipPartMeter meter = Capacity id = Source.ID` or similar.